pkg/console/controllers/clidownloads/controller.go

```diff
@@ -150,16 +150,19 @@ func GetPlatformURL(baseURL, platform, archiveType string) string {
 	return fmt.Sprintf("%s/%s/%s", baseURL, platform, archiveType)
 }
 
-func PlatformBasedOCConsoleCLIDownloads(host, arch, cliDownloadsName string) *v1.ConsoleCLIDownload {
-	baseURL := fmt.Sprintf("%s/%s", util.HTTPS(host), arch)
+func PlatformBasedOCConsoleCLIDownloads(host, cliDownloadsName string) *v1.ConsoleCLIDownload {
+	baseURL := fmt.Sprintf("%s", util.HTTPS(host))
 	platforms := []struct {
 		label    string
 		key      string
 		archType string
 	}{
-		{"Linux", "linux", "oc.tar"},
-		{"Mac", "mac", "oc.zip"},
-		{"Windows", "windows", "oc.zip"},
+		{"Linux for X86-64", "amd64/linux", "oc.tar"},
+		{"Linux for ARM 64", "arm64/linux", "oc.tar"},
+		{"Linux for Power LE", "ppc64le/linux", "oc.tar"},
+		{"Linux for Z", "s390x/linux", "oc.tar"},
+		{"Mac", "amd64/mac", "oc.zip"},
+		{"Windows", "amd64/windows", "oc.zip"},
```

Comment:
I'd prefer if these were "for amd64" or "for x86-64" or whatever Red Hat's preferred name for the architecture is instead of leaving it implicit.  Incidentally, [Wikipedia uses x86-64][1], not X86-64.

[1]: https://en.wikipedia.org/wiki/X86-64

Reply:
By "implicit" are you referring to "Mac" and "Windows"?  My patch does not change the name of those, and doing so might be more appropriate for a separate PR.

Reply:
Repushed with (hopefully) more "offical" names for the Linux architectures (not that there is ever any consistency with these names).

Reply:
> By "implicit" are you referring to "Mac" and "Windows"?

Yup.

> My patch does not change the name of those, and doing so might be more appropriate for a separate PR.

I'm not a maintainer here, but "add qualifying arch names to download links for all existing downloads" seems in-scope for a PR that adds multi-arch support ;).

Reply:
I changed "Windows" to "Windows 64-bit" in a new commit.  I see no use to adding anything to "Mac", as only 64-bit Intel is currently supported.

Reply:
I don't think "64-bit" is sufficiently qualified (seems like it would also include arm64, etc.).  I also think we need to be explicit about Mac, because folks could be running macOS/Darwin on other arches even if we don't support running `oc` on those arches.  I've filed #353 to address both points, PTAL.